hermes-api/src/main/java/pl/allegro/tech/hermes/api/SubscriptionMetrics.java

```diff
@@ -67,9 +95,88 @@ public boolean equals(Object obj) {
         return Objects.equals(this.delivered, other.delivered)
             && Objects.equals(this.discarded, other.discarded)
             && Objects.equals(this.rate, other.rate)
+            && Objects.equals(this.httpStatusCodesM1, other.httpStatusCodesM1)
+            && Objects.equals(this.otherErrorsM1, other.otherErrorsM1)
+            && Objects.equals(this.timeoutsM1, other.timeoutsM1)
             && Objects.equals(this.state, other.state);
     }
 
+    private static class HttpStatusCodeMetrics {
+        public String class1xx;
+        public String class2xx;
+        public String class3xx;
+        public String class4xx;
+        public String class5xx;
+
+        public HttpStatusCodeMetrics() {
+        }
+
+        public HttpStatusCodeMetrics(String class1xx, String class2xx, String class3xx, String class4xx, String class5xx) {
+            this.class1xx = class1xx;
+            this.class2xx = class2xx;
+            this.class3xx = class3xx;
+            this.class4xx = class4xx;
+            this.class5xx = class5xx;
+        }
+
+        public String getClass1xx() {
+            return class1xx;
+        }
+
+        public void setClass1xx(String class1xx) {
+            this.class1xx = class1xx;
+        }
+
+        public String getClass2xx() {
+            return class2xx;
+        }
+
+        public void setClass2xx(String class2xx) {
+            this.class2xx = class2xx;
+        }
+
+        public String getClass3xx() {
+            return class3xx;
+        }
+
+        public void setClass3xx(String class3xx) {
+            this.class3xx = class3xx;
+        }
+
+        public String getClass4xx() {
+            return class4xx;
+        }
+
+        public void setClass4xx(String class4xx) {
+            this.class4xx = class4xx;
+        }
+
+        public String getClass5xx() {
+            return class5xx;
+        }
+
+        public void setClass5xx(String class5xx) {
+            this.class5xx = class5xx;
+        }
+
+        @Override
+        public boolean equals(Object o) {
```

Comment:
same here, i don't think this object is ever used in any hashed collection (or any collection)